integration-tests/mockApis/manageusers.js

```diff
@@ -102,7 +201,7 @@ const stubRoleDetails = ({
   },
 }) =>
   getFor({
-    urlPattern: '/roles/[^/]*',
+    urlPattern: '/roles/(rolecode0|AUTH_GROUP_MANAGER)',
```

Comment:
Wondering if `\/roles\/[^(\/|paged)]*` would be better - it just needs to exclude the roles/paged